Python/books.md

```diff
@@ -1,6 +1,18 @@
-Name | Link
------------- | ------------- 
-Learn Python the Hard Way | http://learnpythonthehardway.org/
-Core Python Programming (2nd Edition) | http://www.amazon.com/Core-Python-Programming-Wesley-Chun/dp/0132269937
-How to Think Like a Computer Scientist: Learning with Python | http://www.lulu.com/shop/allen-downey/how-to-think-like-a-computer-scientist-learning-with-python/paperback/product-3487688.html
-Violent Python | http://shop.oreilly.com/product/9781597499576.do
+### Python Books
+
+* [Learn Python the Hard Way ](http://learnpythonthehardway.org/) - Exercise based step by step guide to basic python programming.
+* [Core Python Programming (2nd Edition)](http://www.amazon.com/Core-Python-Programming-Wesley-Chun/dp/0132269937) - Simple guide on core python concepts like Variable, Loops, Collections etc.
+* [How to Think Like a Computer Scientist: Learning with Python](http://www.lulu.com/shop/allen-downey/how-to-think-like-a-computer-scientist-learning-with-python/paperback/product-3487688.html) - Here you'll find a short introduction to Logo as a general purpose programming language, as opposed to its limited educational use with turtle graphics.
+* [Violent Python ](http://shop.oreilly.com/product/9781597499576.do) - A Cookbook for Hackers, Forensic Analysts, Penetration Testers and Security Engineers.
+* [Core python application programming](https://5d0c00d0babedc9d7abd62f30361d865b480e494.googledrive.com/host/0B4m99Zn8K7eGZzlUTFZCX20yVE0/python/Core%20Python%20Applications%20Programming%203rd.pdf) - Mostly based on Generel programming concept and Web Development.
+* [Learning Python](http://shop.oreilly.com/product/0636920028154.do) - Step by step guide to all basic fundamentals of python programming.
+* [Python Cookbook](http://shop.oreilly.com/product/0636920027072.do) - Introduction to python features and usefull modules.
+* [Python network programming](http://www.amazon.in/Python-Network-Programming-Cookbook-Faruque/dp/1849513465) - Python,Networking.
+* [Python programming for absolute beginner](http://www.amazon.com/Python-Programming-Absolute-Beginner-Edition/dp/1435455002) - For newbies and kid.
+* [Fundamentals of Python From First Programs through Data Structures](http://www.amazon.com/Fundamentals-Python-Programs-Introduction-Programming/dp/1111822700) - Basic to advance level programming with python.
+* [Head first python](http://shop.oreilly.com/product/0636920003434.do) - Python, Step by Step.
+* [Expert python programming](http://www.amazon.com/Expert-Python-Programming-practices-distributing/dp/184719494X) - Advanced level concepts of python.
+* [Think Python](shop.oreilly.com/product/0636920025696.do) - Another fundamental guide to python featuers.

```

Comment:
this is an incomplete URL! When I view your commit, it's redirecting me to https://github.com/PSanni/Learning-Resources/blob/PythonBooks/Python/shop.oreilly.com/product/0636920025696.doa